parachain/runtime/src/lib.rs

```diff
@@ -368,7 +368,7 @@ impl bridge::Config for Runtime {
 
 #[cfg(not(feature = "test-e2e"))]
 parameter_types! {
-	pub const DescendantsUntilFinalized: u8 = 35;
+	pub const DescendantsUntilFinalized: u8 = 3;
```

Comment:
This is quite dangerous for mainnet I think. But as a temporary thing, that's fine

Reply:
yeah is a total temporary thing, as ropsten is our de-facto "live" environment until we actually deploy a production stack connected to mainnet

Reply:
Can we add a ticket for that though?

Reply:
Done actually: https://github.com/Snowfork/polkadot-ethereum/issues/291